keps/sig-node/2400-node-swap/README.md

```diff
@@ -205,6 +200,7 @@ will be necessary to implement the third scenario.
 - Supporting zram, zswap, or other memory types like SGX EPC. These could be
   addressed in a follow-up KEP, and are out of scope.
 - Use of swap for cgroupsv1.
+- Handling evictions, scheduling or pod-level APIs.
```

Comment:
This should be made more clear.

Swap is not used for eviction decisions but evictions can still work?

Reply:
You're right, that's unclear.
I've tried to clarify it now, let me know what you think!

Reply:
ty!